bower.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ramda.github.io",
+  "version": "0.14.0",
+  "homepage": "https://ramdajs.com",
+  "authors": [
+    "(Scott Sauyet <scott@sauyet.com>)",
+    "(Michael Hurley <mh@buzzdecafe.com>)"
+  ],
+  "description": "A practical functional library for Javascript programmers.",
+  "moduleType": [
+    "amd",
+    "globals",
+    "node"
+  ],
+  "keywords": [
+    "functional"
+  ],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests",
+    "examples",
+    "logo"
+  ],
+  "devDependencies": {
+    "bootstrap": "~3.3.1"
+  }
+}
```

Comment:
i think we need it for the build:

ramda.less: `@import "bower_components/bootstrap/less/bootstrap"`
variables.less: `@icon-font-path: "fonts/";`

I suggest we table this. We can come back and improve on it later.